src/features/users/Account.tsx

```
@@ -0,0 +1,174 @@
+                floatingLabel={true}
+                autoComplete="new-password"
+                placeholder="Current password"
+                fullWidth
+                {...register('password', {
+                  required: 'Password is required',
+                  onChange: async () => {
+                    await trigger(['password']);
+                  },
+                })}
+                state={formErrors.password ? 'error' : 'normal'}
+                helperText={(formErrors.password && formErrors.password.message) || ''}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)